cli/fmt.rs

```diff
@@ -9,10 +9,10 @@
 
 use crate::colors;
 use crate::diff::diff;
+use crate::dprint_plugin_typescript as dprint;
```

Comment:
where does it come from if not from a crate dependency? I'm confused.

Reply:
deno_lint re-exports it

Reply:
Does deno_lint actually use dprint_plugin_typescript? It doesn't seem like it.